examples/classifiers/ws_gru_mesa.py

```diff
@@ -0,0 +1,118 @@
+# %%
+from tensorflow.keras import layers, models
+from tqdm import tqdm
+
+from sleepecg import (
+    evaluate,
+    extract_features,
+    load_classifier,
+    prepare_data_keras,
+    print_class_balance,
+    read_mesa,
+    read_shhs,
+    save_classifier,
+)
+
+# %% Read data and extract features
+records = list(read_mesa())
+
+features_train, stages_train, feature_ids = extract_features(
+    tqdm(records),
+    lookback=240,
+    lookforward=270,
+    feature_selection=[
+        'hrv-time',
+        'hrv-frequency',
+        'recording_start_time',
+        'age',
+        'gender',
+    ],
+    min_rri=0.3,
+    max_rri=2,
+    max_nans=0.5,
+)
+
+# %% Merge sleep stages, pad and mask data as preparation for keras NN
+stages_mode = 'wake-sleep'
+
+features_train_pad, stages_train_pad, sample_weight = prepare_data_keras(
+    features_train,
+    stages_train,
+    stages_mode,
+)
+print_class_balance(stages_train_pad, stages_mode)
+
+# %% Define and train model
+model = models.Sequential([
+    layers.Input((None, features_train_pad.shape[2])),
+    layers.Masking(-1),
+    layers.BatchNormalization(),
+
+    layers.Dense(64),
+    layers.ReLU(),
+    layers.Bidirectional(layers.GRU(8, return_sequences=True)),
+    layers.Bidirectional(layers.GRU(8, return_sequences=True)),
+
+    layers.Dense(stages_train_pad.shape[-1], activation='softmax'),
+])
+
+model.compile(
+    optimizer='rmsprop',
+    loss='categorical_crossentropy',
+    metrics=['accuracy'],
+)
+model.build()
+model.summary()
+
+# %% Train model
+model.fit(
+    features_train_pad,
+    stages_train_pad,
+    epochs=5,
+    sample_weight=sample_weight,
+)
+
+# %% Store classifier
+save_classifier(
+    name='ws-gru-mesa',
```

Comment:
I think we should explain the name. `ws` is wake/sleep, `mesa` is also clear, but what does `gru` mean? This should also mentioned in the docs.

Reply:
The table on the [doc page](https://sleepecg--78.org.readthedocs.build/en/78/classification.html#included-classifiers) contains a link to the wikipedia page of [gated recurrent units].

We could add a comment like this?
```
# Classifier name explanation:
# ws -> WAKE-SLEEP
# gru -> main component of the classifier network, a gated recurrent unit
# mesa -> MESA dataset (https://sleepdata.org/datasets/mesa)
```

Reply:
I didn't see this link, I think it's fine then, no need to explain further.